README.md

```diff
@@ -149,6 +149,7 @@ Hurray! You successfully made a contribution!
   - [Alex Valdez](https://github.com/arevaldez)
   - [Ali Mirrajei](https://github.com/alimirrajei)
   - [Alpadmin](https://github.com/hasthamalp/)
+  - [Aman Arora (she/her)](https://github.com/sheisAman)
```

Comment:
Maybe only provide your name and remove the pronouns?
```suggestion
  - [Aman Arora](https://github.com/sheisAman)
```

Reply:
ahmm, ok, but how adding an extra word could affect?

Reply:
It doesn't affect anything, but IMO it looks better without the pronouns. Just a suggestion, you can keep it if you want. 🙂